hack/choco/update-choco-package.sh

```diff
@@ -54,11 +59,14 @@ rm -fv hack/choco/tools/chocolateyinstall.ps1-e
 version="${version:1}"
 sed -i -e "s/\(<version>\).*\(<\/version>\)/<version>""${version}""\<\/version>/g" hack/choco/tanzu-community-edition.nuspec
 rm -fv hack/choco/tanzu-community-edition.nuspec-e
- 
+
+sed -i -e "s/\(\$checksum64 =\).*/\$checksum64 = ""'${windows_amd64_shasum}'""/g" hack/choco/tools/chocolateyinstall.ps1 
```

Comment:
`""'${windows_amd64_shasum}'""` looks strange to me - should we just use `'${windows_amd64_shasum}'`? Is there something I'm missing with interpolation in the double quotes?

Reply:
or just escaping `\"` should work (i think)

Reply:
Looking at the powershell scripts, they use `'`. However, I think with discussion on #3242 means this file will be removed and replaced by a powershell script on Windows workers.

Reply:
Yes `\` can also work will use that.